src/commands/apps/builds/logs.ts

```diff
@@ -0,0 +1,142 @@
+import appBuildsService from '@/services/app-builds.js';
+import appsService from '@/services/apps.js';
+import authorizationService from '@/services/authorization-service.js';
+import organizationsService from '@/services/organizations.js';
+import { unescapeAnsi } from '@/utils/ansi.js';
+import { prompt } from '@/utils/prompt.js';
+import { wait } from '@/utils/wait.js';
+import { defineCommand, defineOptions } from '@robingenz/zli';
+import consola from 'consola';
+import { hasTTY } from 'std-env';
+import { z } from 'zod';
+
+function formatDate(date: Date): string {
```

Comment:
I'm not sure right now if we print timestamps in the apps:builds:create command. Let us implement it the same way.